example_configs/seafile.md

```diff
@@ -23,6 +25,34 @@ FILTER = memberOf=cn=seafile_user,ou=groups,dc=example,dc=com
 ```
 * Replace `seafile_user` with the name of your group.
 
+## Configuring Seafile v11.0+ to use LLDAP directly
```

Comment:
Let's move this section to the top: we want to encourage people to use up to date software (especially in this case)

Reply:
You're absolutely right, done.